app/static/src/app/store/run/run.ts

```diff
@@ -15,12 +16,15 @@ export const defaultState: RunState = {
     resultDiscrete: null,
     userDownloadFileName: "",
     downloading: false,
-    numberOfReplicates: 5
+    numberOfReplicates: 5,
+    parameterSets: [],
```

Comment:
at some point perhaps we'll group these - feels like we've got two bits of data that are strongly enough linked that this might get annoying? Could be solved by making ParameterSet hold its own result perhaps? But you may have already thought of that and ruled it out for some reason?

Reply:
I think I probably just did it this way to make the sets indexable by name - but actually you're usually iterating through the sets anyway, so if the results were in the set they'd be immediately accessible then anyway. Agreed that having too many properties at the top level is best avoided. I'll make a maintenance ticket for that change, or we could just refactor next time we need to put in any parameter set related prop.

Reply:
I suppose the other reason was to have consistency in runState and sensitivityState both having `parameterSetResults` properties.